index.html

```diff
@@ -70,7 +70,7 @@ <h2>Tonic</h2>
     
             <div class="projects">
                 <div class="project-img project-right-img">
-                    <img class="project-img" src="images/second project.svg" alt="">
```

Comment:
:white_check_mark: Image text alternatives** (img alt="alt text") (We added appropiate alt texts to every image)